ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5257,3 +5260,328 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: createAsset
+    uris: "{{ asset_prefix }}/v1/create"
+    upstream_url: "{{ content_service_url }}/asset/v4/create"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: updateAsset
+    uris: "{{ asset_prefix }}/v1/update"
+    upstream_url: "{{ content_service_url }}/asset/v4/update"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```

Comment:
ACL need to be contentUpdate @rheafernandes

Reply:
done